dev-python/pandas/pandas-0.23.3.recipe

```diff
@@ -37,26 +38,70 @@ BUILD_REQUIRES="
 	pytz_python3
 	"
 BUILD_PREREQUIRES="
-	cmd:python
-	cmd:python3
-    cmd:gcc
-	cmd:gfortran
+	cmd:gcc$secondaryArchSuffix
+	cmd:gfortran$secondaryArchSuffix
+	cmd:pkg_config$secondaryArchSuffix
 	"
 
-PYTHON_PACKAGES=(python python3)
-PYTHON_VERSIONS=(2.7 3.6)
+
+PYTHON_PACKAGES=()
+PYTHON_VERSIONS=()
+PYTHON_LIBSUFFIXES=()
+# We don't have python2 for secondaryArch
+if [ -z "$secondaryArchSuffix" ]; then
+	PYTHON_PACKAGES+=(python)
+	PYTHON_VERSIONS+=(2.7)
+	PYTHON_LIBSUFFIXES+=("")
+	BUILD_REQUIRES+="
+		setuptools_python
+		cython_python
+		python_numpy
+		dateutil_python
+		pytz_python
+		"
+	BUILD_PREREQUIRES+="
+		cmd:python2
+		"
+fi
+# gcc2 does not support the flags passed by python3
+if [ "$effectiveTargetArchitecture" != x86_gcc2 ]; then
+	PYTHON_PACKAGES+=(python3)
+	PYTHON_VERSIONS+=(3.6)
+	PYTHON_LIBSUFFIXES+=(m)
+	BUILD_REQUIRES+="
+		setuptools_python3
+		cython_python3
+		python3_numpy
+		dateutil_python3
+		pytz_python3
+		"
+	BUILD_PREREQUIRES+="
+		cmd:python3
+		"
+fi
 for i in "${!PYTHON_PACKAGES[@]}"; do
 	pythonPackage=${PYTHON_PACKAGES[i]}
 	pythonVersion=${PYTHON_VERSIONS[$i]}
+	pythonLibSuffix=${PYTHON_LIBSUFFIXES[$i]}
 
 	eval "PROVIDES_$pythonPackage=\"
 		${portName}_$pythonPackage = $portVersion
 		\""
 	eval "REQUIRES_$pythonPackage=\"
-		haiku
+		haiku$secondaryArchSuffix
 		cmd:python$pythonVersion
+		setuptools_$pythonPackage
+		cython_$pythonPackage
+		$pythonPackage_numpy
```

Comment:
```diff
-		cython_$pythonPackage
+		cython_$pythonPackage$secondaryArchSuffix
-		$pythonPackage_numpy
+		${pythonPackage}_numpy$secondaryArchSuffix
```
Oops, I just noticed that `python3_numpy` does not have support for x86 secondary arch.

So what you can do is apply the above changes and mark `x86` as untested (`?x86`) in `SECONDARY_ARCHITECTURES`.
If someone adds support for x86 secondary arch to `python3_numpy` then `hp pandas_x86` will work (and create `pandas_x86_python3`.)
Remark: merging `python_numpy` and `python3_numpy` would probably be a good idea.

Reply:
@fbrosson Fixed some things :)
Didn't understand what merging `python_numpy` and `python3_numpy` meant, though...

Reply:
Merging `python_numpy` and `python3_numpy` means that you delete `python_numpy-1.14.5.recipe` and `python3_numpy-1.14.5.recipe` and create a single recipe, `numpy-1.14.5.recipe`, with support for x86 secondary arch, where you would have:
```
PROVIDES="
	numpty$secondaryArchSuffix = $portVersion
	"
```
and a similar layout for the _python and _python3 subpackages as the one you have for e.g. this recipe or cryptography.

So, on x86_64, instead of having `python_numpy` and `python3_numpy` you would have `numpy_python` and `numpy_python3` (and an almost empty `numpy` base package.)

And on x86_gcc2h, instead of having `python_numpy` and `python3_numpy_x86` you would have `numpy_python` (and an almost empty `numpy` base package) as well as  `numpy_x86_python3` (and an almost empty `numpy_x86` base package.)

BTW, `cython_python-0.28.2.recipe` and `cython_python3-0.28.2.recipe` might also be merged into a single recipe: `cython-0.28.2.recipe`.

**Warning:** it looks like `cython_python3_x86` is failing to build (on x86_gcc2h), so, in case you would like to try a merge then you will need to be prepared to spend a lot of time on it... 

**Last but not least:** there might be a valid reason why the recipes for cython and numpy were not merged. So you should probably ask for a green light (from a few key contributors) if you plan to  work on these merges.